app/model_status.py

```diff
@@ -86,20 +87,14 @@ def _build_status_dict(self) -> Dict[str, Any]:
                 app.config['DESIRED_CLASSIFIER_PRECISION'], {})
             for t, ti in self.classifier.topic_infos.items():
                 topics[t] = {
-                    'name':
-                        t,
-                    'samples':
-                        ti.num_samples,
-                    'quality':
-                        ti.recalls.get(
-                            app.config['DESIRED_CLASSIFIER_PRECISION'], 0.0),
+                    'name': t,
+                    'samples': ti.num_samples,
+                    'quality': ti.get_quality(),
                 }
         return {
             'name': self.model_name,
             'instances': instances,
             'preferred': preferred,
-            'completeness': quality_at_precision.get('completeness', 0.0),
```

Comment:
Please keep these - they are the only way I personally understand how to judge the overall quality of the model...